.travis.yml

```diff
@@ -1 +1,10 @@
 language: node_js
+services: redis-server
+env: DB=postgres
+before_script: psql -c 'create database testingdatabase;' -U postgres && sleep 1 && psql -U postgres -d testingdatabase -a -f tools/pg/sbcontent.sql && sleep 1 && psql -U postgres -d testingdatabase -a -f tools/pg/sbentity.sql
+
+addons:
+ postgresql: "9.3"
+
+notifications:
+ email: false
```

Comment:
Yes, these should replace the nightly builds we are currently doing, and send emails.